cockatrice/src/handle_public_servers.h

```diff
@@ -0,0 +1,37 @@
+#ifndef COCKATRICE_HANDLE_PUBLIC_SERVERS_H
+#define COCKATRICE_HANDLE_PUBLIC_SERVERS_H
+
+#include "userconnection_information.h"
+#include <QNetworkReply>
```

Comment:
More a hint than a comment.
Since this header file is meant to be included in other classes (eg. dlg_connect.cpp), and `#include` basically means "replace this line with the whole content of the included file", you may want to minimize the number of `#include`s that gets propagated.

In this case, since here you are not using the `QNetworkReply` class itself but only a pointer to it, you can just replace this line with a dummy
```
class QNetworkReply;
class QNetworkAccessManager;
```

and move the `#include <QNetworkReply>` in the `handle_public_servers.cpp` file, where the full class definition is really needed.

Using this hint everywhere in he code can speed up compilation a lot and also, in some rare cases, avoid obscure bugs due to double inclusion/definition of the same named resource.

Reply:
I see, never really understood the purpose. Thanks!